### PR TITLE
quarto: refresh page

### PR DESCRIPTION
**Pull request by PMassicotte**
Updating the examples and adding more options to the use of the `quarto` command.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/quarto.md [hunk 1]
```diff
@@ -3,34 +3,22 @@
...
+- Create a new blog website:
 
-`quarto create-project {{path/to/destination_directory}} --type {{website}}`
+`quarto create-project {{path/to/destination_directory}} --type {{website}} --template {{blog}}`
 
-- Create a new book project:
+- Render input file(s) to different formats:
 
-`quarto create-project {{path/to/destination_directory}} --type {{book}}`
+`quarto render {{path/to/file.qmd[.rmd|.ipynb]}} --to {{html|pdf|docx}}`
```
Comment (CleanMachine1) [inline pages/common/quarto.md:16]:
```suggestion
`quarto render {{path/to/file.qmd|rmd|ipynb]}} --to {{html|pdf|docx}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, after @CleanMachine1's suggestion. Thanks for your contribution.

File: pages/common/quarto.md [hunk 2]
```diff
@@ -3,34 +3,22 @@
...
+- Create a new blog website:
 
-`quarto create-project {{path/to/destination_directory}} --type {{website}}`
+`quarto create-project {{path/to/destination_directory}} --type {{website}} --template {{blog}}`
 
-- Create a new book project:
+- Render input file(s) to different formats:
 
-`quarto create-project {{path/to/destination_directory}} --type {{book}}`
+`quarto render {{path/to/file.qmd[rmd|ipynb]}} --to {{html|pdf|docx}}`
```
Comment (CleanMachine1) [inline pages/common/quarto.md:16]:
```suggestion
`quarto render {{path/to/file.qmd|rmd|ipynb]}} --to {{html|pdf|docx}}`
```

File: pages/common/quarto.md [hunk 3]
```diff
@@ -3,34 +3,22 @@
...
+- Create a new blog website:
 
-`quarto create-project {{path/to/destination_directory}} --type {{website}}`
+`quarto create-project {{path/to/destination_directory}} --type {{website}} --template {{blog}}`
 
-- Create a new book project:
+- Render input file(s) to different formats:
 
-`quarto create-project {{path/to/destination_directory}} --type {{book}}`
+`quarto render {{path/to/file.qmd|rmd|ipynb]}} --to {{html|pdf|docx}}`
```
Comment (kbdharun) [inline pages/common/quarto.md:16]:
```suggestion
`quarto render {{path/to/file.qmd|rmd|ipynb}} --to {{html|pdf|docx}}`
```

Comment (PMassicotte) [reply pages/common/quarto.md:16]: Thank you!

Comment (PMassicotte) [reply pages/common/quarto.md:16]: What is the proper way to indicate multiple choices? I based my suggestion on the `tar` entry:

image

Comment (PMassicotte) [reply pages/common/quarto.md:16]: I think it should read `quarto render {{path/to/file.qmd[.rmd|.ipynb]}} --to {{html|pdf|docx}}`

Comment (CleanMachine1) [reply pages/common/quarto.md:16]: We could use 
```suggestion
`quarto render {{path/to/file.{{qmd|rmd|ipynb}}}} --to {{html|pdf|docx}}`
```
But this wont change the appearance of how it renders, but your suggestion @PMassicotte doesn't match our syntax

Comment (PMassicotte) [reply pages/common/quarto.md]: All good! Thank you for helping.

**Review by CleanMachine1 — APPROVED** [review]
Thanks for your contribution

**Pull request merged**